### PR TITLE
add eBee name

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <bill@isovalent.com>

### CODE REVIEW

File: src/pages/what-is-ebpf.js [hunk 1]
```diff
@@ -94,6 +94,12 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first 
+	    <a href="https://ebpf.io/summit-2020"> eBPF Summit </a>
```
Comment (dthaler) [inline src/pages/what-is-ebpf.js:100]:
```suggestion
	    <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
```

File: src/pages/what-is-ebpf.js [hunk 2]
```diff
@@ -94,6 +94,12 @@ class Description extends React.Component {
...
+	    there was a vote taken and the bee was named eBee.
```
Comment (qmonnet) [inline src/pages/what-is-ebpf.js:101]:
```suggestion
	    there was a vote taken and the bee was named **eBee**.
```

**Review by qmonnet — APPROVED** [review]
Would this be a chance to provide guidance on how to reuse the logo (license, acceptable use cases...)?

**Comment by dthaler** [comment]
> Would this be a chance to provide guidance on how to reuse the logo (license, acceptable use cases...)?

Yes.  Thomas Graf has had that action item since the 2021-10-13 BSC meeting as noted in the [minutes](https://github.com/ebpf-io/bsc/blob/master/minutes.md)

File: src/pages/what-is-ebpf.js [hunk 3]
```diff
@@ -94,6 +94,12 @@ class Description extends React.Component {
...
+          </p>
```
Comment (xmulligan) [inline src/pages/what-is-ebpf.js:102]:
```suggestion
          For details acceptable uses of the logo, please see our
          <a href="https://ebpf.io/brand-guidelines/"> Brand Guidelines </a>
          </p>
```

File: src/pages/what-is-ebpf.js [hunk 4]
```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
...
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first 
+	    <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
+	    there was a vote taken and the bee was named **eBee**.
+          For details acceptable uses of the logo, please see our
+          <a href="https://ebpf.io/brand-guidelines/"> Brand Guidelines </a>
+          </p>
```
Comment (xmulligan) [inline src/pages/what-is-ebpf.js:104]:
```suggestion
            The bee is the official logo for eBPF. At the first 
	    <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
	    there was a vote taken and the bee was named **eBee**.
	    For details acceptable uses of the logo, please see our
	    <a href="https://ebpf.io/brand-guidelines/"> Brand Guidelines </a>
          </p>
```

**Comment by xmulligan** [comment]
I've added a link to the brand guidelines page, but obviously it is not built out yet. Let me see if I can work on that

File: src/pages/what-is-ebpf.js [hunk 5]
```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first 
+	    <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
+	    there was a vote taken and the bee was named **eBee**.
+	    For details acceptable uses of the logo, please see our
+	    <a href="https://ebpf.io/brand-guidelines/"> Brand Guidelines </a>
```
Comment (qmonnet) [inline src/pages/what-is-ebpf.js:103]: Looks like the file is using spaces for indent.

```suggestion
            <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
            there was a vote taken and the bee was named **eBee**.
            (For details on acceptable uses of the logo, please see our
            <a href="https://ebpf.io/brand-guidelines/">Brand Guidelines</a>.)
```

**Review by qmonnet — CHANGES_REQUESTED** [review]
Thanks for addressing my comments! But do you plan to publish the guidelines soon? If not, I'd rather have this addition as a follow-up and avoid introducing a broken link (Alternatively, we could add a “Coming Soon” placeholder at ebpf.io/brand-guidelines, to show that this is not simply a mistake).

File: src/pages/what-is-ebpf.js [hunk 6]
```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first 
+	    <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
+	    there was a vote taken and the bee was named **eBee**.
```
Comment (qmonnet) [inline src/pages/what-is-ebpf.js:101]: Oh and sorry I'm writing too much Markdown :facepalm: 

```suggestion
	    there was a vote taken and the bee was named <b>eBee</b>.
```

**Comment by dthaler** [comment]
> I've added a link to the brand guidelines page, but obviously it is not built out yet. Let me see if I can work on that

Thanks!   I think we should build it out before merging a link to it though so we don't add dead links.

**Comment by xmulligan** [comment]
I've created a draft of the branding page here https://docs.google.com/document/d/1Ln0Rd-m-Ouw4fjD38KH_L_BfHEgbTyv4ESHCytIX8P0/edit 

Let me know if you have any comments and I can get the page made

**Comment by dthaler** [comment]
> I've created a draft of the branding page here https://docs.google.com/document/d/1Ln0Rd-m-Ouw4fjD38KH_L_BfHEgbTyv4ESHCytIX8P0/edit
> 
> Let me know if you have any comments and I can get the page made

Looks good, let's get a page made.  Regarding "Please don’t use our ... logos ... in ways that may ... suggest our sponsorship, endorsement, or affiliation": currently https://events.linuxfoundation.org/cloud-native-ebpf-day-europe/ uses the ebpf logo but I cannot find in any eBPF Foundation governing board or BSC meeting minutes that the foundation explicitly authorized this.  Thomas and I are on the program committee and I did get Lindsay to fix the colors to match the brand guidelines.  But do we need an explicit GB or BSC action item to ratify this?   Who in the Foundation is authorized to "endorse" such use, is it the Governing Board?  The BSC?  A Legal Committee?  The chair of the board? Please add the answer somewhere.

File: src/pages/what-is-ebpf.js [hunk 7]
```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first 
+            <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
+            there was a vote taken and the bee was named **eBee**.
+            (For details on acceptable uses of the logo, please see our
+            <a href="https://ebpf.io/brand-guidelines/">Brand Guidelines</a>.)
```
Comment (xmulligan) [inline src/pages/what-is-ebpf.js:103]:
```suggestion
            (For details on acceptable uses of the logo, please see the Linux Foundation guidelines
            <a href="https://linuxfoundation.org/brand-guidelines/">Brand Guidelines</a>.)
```

**Comment by qmonnet** [comment]
Please fix the formatting and the spacing around the links
image

File: src/pages/what-is-ebpf.js [hunk 8]
```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first 
+            <a href="https://ebpf.io/summit-2020">eBPF Summit</a>
+            there was a vote taken and the bee was named **eBee**.
+            (For details on acceptable uses of the logo, please see the Linux Foundation guidelines
+            <a href="https://linuxfoundation.org/brand-guidelines/">Brand Guidelines</a>.)
```
Comment (xmulligan) [inline src/pages/what-is-ebpf.js:103]:
```suggestion
            The bee is the official logo for eBPF. At the first  
            <a href="https://ebpf.io/summit-2020">eBPF Summit</a> 
            there was a vote taken and the bee was named **eBee**.
            (For details on acceptable uses of the logo, please see the Linux Foundation 
            <a href="https://linuxfoundation.org/brand-guidelines/">Brand Guidelines</a>.)
```

File: src/pages/what-is-ebpf.js [hunk 9]
```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first  
+            <a href="https://ebpf.io/summit-2020">eBPF Summit</a> 
+            there was a vote taken and the bee was named **eBee**.
+            (For details on acceptable uses of the logo, please see the Linux Foundation 
+            <a href="https://linuxfoundation.org/brand-guidelines/">Brand Guidelines</a>.)
```
Comment (xmulligan) [inline src/pages/what-is-ebpf.js:103]:
```suggestion
            The bee is the official logo for eBPF. At the first &nbsp 
            <a href="https://ebpf.io/summit-2020">eBPF Summit</a> &nbsp
            there was a vote taken and the bee was named <b>eBee</b>.
            (For details on acceptable uses of the logo, please see the Linux Foundation &nbsp 
            <a href="https://linuxfoundation.org/brand-guidelines/">Brand Guidelines</a>.) 
```

File: src/pages/what-is-ebpf.js (hunk 8 above)
Comment (qmonnet) [reply src/pages/what-is-ebpf.js]: I'm not sure of how this works under the hood, but it looks like spaces at the end or the beginning of the line are trimmed. Maybe try having the link on the same line as some other text, with no line space between e.g. “first” and `<a href=""...`

File: src/pages/what-is-ebpf.js [hunk 10]
```diff
@@ -94,6 +94,14 @@ class Description extends React.Component {
             becoming contributors to major eBPF projects, eBPF.io will help you
             along the way.
           </p>
+          <h3>What is the bee named?</h3>
+          <p>
+            The bee is the official logo for eBPF. At the first &nbsp 
+            <a href="https://ebpf.io/summit-2020">eBPF Summit</a> &nbsp
+            there was a vote taken and the bee was named <b>eBee</b>.
+            (For details on acceptable uses of the logo, please see the Linux Foundation &nbsp 
+            <a href="https://linuxfoundation.org/brand-guidelines/">Brand Guidelines</a>.)
```
Comment (xmulligan) [inline src/pages/what-is-ebpf.js:103]:
```suggestion
            The bee is the official logo for eBPF. At the first eBPF Summit
            there was a vote taken and the bee was named <b>eBee</b>.
            (For details on acceptable uses of the logo, please see the 
            Linux Foundation <a href="https://linuxfoundation.org/brand-guidelines/">Brand Guidelines</a>.) 
```

**Review by qmonnet — APPROVED** [review]
Thanks!!

**Review by dthaler — APPROVED** [review]
The brand guidelines link content isn't super relevant but I guess it's the best there is so far, so approving.

**Comment by xmulligan** [comment]
Thanks @dthaler !

**Pull request merged**